index.html

```diff
@@ -54,6 +54,32 @@
         Features Registry Repository</a> on GitHub to be notified of changes.
       </p>
     </section>
+    <section>
+      <h2>
+        Change Process
+      </h2>
+      <p>
+        The process for adding and/or updating a [=powerful feature=] in this registry is as
+        follows:
+      </p>
+      <ol>
+        <li>Determine if your feature meets the definition of a [=powerful feature=] (i.e.,
+        requires [=express permission=] to be used).
+        </li>
+        <li>[=Specify a powerful feature=] in your specification in conformance with the
```

Comment:
Is respec smart enough to link `<dfn class="export">specifies a powerful feature</dfn>` once that lands? Or do we need to tweak linking text?